### PR TITLE
Simplify and clean up tasks

### PR DESCRIPTION
**Pull request by ashchan**
- [x] Merge/clean up start scripts so we can use a single task script to get the whole app (both `neuron-wallet` & `neuron-ui` at the moment) up and running. Do not have `run`, `watch`, `start` etc.
  - [x] The react app (`neuron-ui`) should be able to watch and refresh automatically.
  - [x] It's **OK** if the electron app (`neuron-wallet`) cannot hot load or run in watch mode. 
  - [x] Ideally to only have `start` and `start:ui` to run the whole app and the react UI app only respectively.
- [x] Make sure binary packaging still works.
- [x] Make sure every one understand that changing the start script has risk to break packaging, running the app fully.
- [x] Make browser for `neuron-ui` to open by default.

### CODE REVIEW

File: packages/neuron-ui/package.json [hunk 1]
```diff
@@ -12,10 +12,10 @@
   "license": "MIT",
   "scripts": {
     "start": "cross-env BROWSER=none react-app-rewired start",
-    "build": "react-app-rewired build",
-    "test": "react-app-rewired test --env=jsdom --color",
-    "eject": "react-scripts eject",
```
Comment (ashchan) [inline packages/neuron-ui/package.json]: Remove `eject` to avoid anyone running and ejecting the CRA app accidentally.

**Comment by ashchan** [comment]
@Keith-CY Didn't bother making the electron run in watch mode. Do that later when we have free time.

File: packages/neuron-wallet/package.json [hunk 2]
```diff
@@ -13,18 +13,17 @@
   "main": "dist/main.js",
   "license": "MIT",
   "scripts": {
-    "start": "tsc && electron .",
+    "start": "cross-env NODE_ENV=production electron .",
+    "start:dev": "tsc && cross-env NODE_ENV=production electron .",
```
Comment (Keith-CY) [inline packages/neuron-wallet/package.json]: We've used `isPackaged` to distinguish build for dev or prod.

Comment (ashchan) [reply packages/neuron-wallet/package.json]: Hold on a second, will update this.

File: packages/neuron-wallet/src/main.ts [hunk 3]
```diff
@@ -28,15 +25,19 @@ function createWindow() {
...
+      devTools: isDevelopmentMode,
     },
   })
 
   windowState.manage(mainWindow)
 
   Menu.setApplicationMenu(menu)
 
-  mainWindow.loadURL(app.isPackaged ? ENTRY.PROD : ENTRY.DEV)
+  const ENTRY = {
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/main.ts]: `global` variables usually defined at the top

Comment (ashchan) [reply packages/neuron-wallet/src/main.ts]: I'd prefer to move it to where it's used (if there's only one place that uses it).

Comment (ashchan) [reply packages/neuron-wallet/src/main.ts]: In fact in this case, I believe we should simply delete `ENTRY`, and use the two URLs directly.

Comment (Keith-CY) [reply packages/neuron-wallet/src/main.ts]: OK, make it simple for now.

**Pull request merged**